crates/autopilot/src/arguments.rs

```diff
@@ -133,6 +133,10 @@ pub struct Arguments {
 
     #[clap(long, env, default_value = "0")]
     pub limit_order_price_factor: f64,
+
+    // Enable background quoting for limit orders.
+    #[clap(long, env, default_value = "true")]
+    pub enable_limit_order_quoter: bool,
```

Comment:
This argument also should also be added to the `Display` implementation of the `Arguments` struct.

Reply:
Since this error keeps happening we could look into ditching the manual `Display` implementation and instead use the `derivative` crate to only overwrite the formatting implementation of secrets?

Reply:
I had suggested this previously as well. The (IMO valid) argument that @vkgnosis gave is that this is, in theory, a `Display` implementation and not a `Debug` one. Maybe there is some convenient `derive` macro that can help us here though.

Maybe even `clap` has something built-in to display the command line arguments in a nice way.

Reply:
If we find a good way to derive Display then I'm all for that. When I made the PR that moved away from Debug the reasoning was iirc:
- Debug shouldn't hide anything because then you lose the way to actually debug print
- We have a specific goals of what and how we want to display when we print args on startup. We were just abusing Debug because it conveniently derived all fields.

Reply:
I tried looking for crates which make deriving the `Display` trait easier but I didn't find anything nice.
If we are willing to experiment a bit we could see if we can accomplish our goals by using wrapper types. For example I tried to come up with a type `Secret<T>` which behaves like a `T` but prints `SECRET` when printed with `Display` formatting. It's still unclear how we could make that interact nicely with lists and options, though.

```rust
#[derive(Clone)]
pub struct Secret<T: Sized>(pub T);

impl<T> std::fmt::Display for Secret<T> {
    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
        writeln!(f, "SECRET",)?;
        Ok(())
    }
}
impl<T> std::ops::Deref for Secret<T> {
    type Target = T;
    fn deref(&self) -> &Self::Target {
        &self.0
    }
}
impl<T: std::fmt::Debug> std::fmt::Debug for Secret<T> {
    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
        self.0.fmt(f)
    }
}
impl<T: std::str::FromStr> std::str::FromStr for Secret<T> {
    type Err = <T as std::str::FromStr>::Err;
    fn from_str(src: &str) -> Result<Self, Self::Err> {
        Ok(Secret(T::from_str(src)?))
    }
}
```

Reply:
Either way, I do think this change would be out of the scope of this PR.